src/oscar/core/utils.py

```diff
@@ -39,9 +44,11 @@ def slugify(value):
         slugifier = import_string(slugifier)
 
     # Use unidecode to convert non-ASCII strings to ASCII equivalents where
-    # possible.
-    value = slugifier(unidecode(six.text_type(value)))
+    # possible if unicode is not allowed to contain in slug.
+    if not getattr(settings, 'OSCAR_SLUG_ALLOW_UNICODE', None):
```

Comment:
The `settings.OSCAR_SLUG_ALLOW_UNICODE` should always be set right? So better to use it directly (also on the other places)